front/lib/plans/subscription.ts

```diff
@@ -318,9 +307,9 @@ export const pokeUpgradeWorkspaceToPlan = async (
     );
   }
 
-  if (newPlan.stripeProductId !== null) {
+  if (newPlan.code === PRO_PLAN_SEAT_29_CODE) {
     throw new Error(
-      `Cannot subscribe to plan ${planCode}: stripeProductId is not null.`
+      `Cannot subscribe to plan ${planCode}: it is the pro plan which requires a stripe subscription.`
```

Comment:
NIT: "Pro plan subscription can only be done through a Stripe checkout session on the app"